### DISCUSSION TITLE
Input file for PHOEBE

### DISCUSSION THREAD

**Discussion by kittipongwangnok** [phoebe-project/phoebe2#950] [General]
Based on the publication I found so far from NASA ADS, I just wondering, can we change the input of the y-axis from flux to magnitude to model? Will it reflect the fitting parameters?

**Comment by kecnry** [comment]
See #507 for a detailed discussion on this (and why PHOEBE does not currently have support for magnitudes - although you can/should convert your data to fluxes when importing into PHOEBE).
